### PR TITLE
Upgrading to api-compiler:0.0.5

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
Fixes https://github.com/googleapis/toolkit/issues/144

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -54,13 +54,13 @@ ext {
 
     // Gapi
     // TODO(cbao): Use released version instead of snapshot.
```
Comment (geigerj) [inline build.gradle]: Can we now remove this TODO?

**Comment by geigerj** [comment]
LGTM

File: build.gradle (hunk 1 above)
Comment (garrettjonesgoogle) [inline build.gradle]: oh of course!

**Comment by bjwatson** [comment]
LGTM

There's still a little left in https://github.com/googleapis/toolkit/pull/284 that could be useful, like using the official Nexus server and excluding the `src-gen/` directory.

**Comment by garrettjonesgoogle** [comment]
I'll switch to the official Nexus server in a following PR.

**Comment by bjwatson** [comment]
SGTM. I commented on a couple other things in https://github.com/googleapis/toolkit/pull/284 that could be valuable to pull into a PR.

**Pull request merged**